sources/_sector/dex/trades/optimism/_sources.yml

```diff
@@ -2,6 +2,8 @@ version: 2
 
 sources:
   - name: uniswap_v3_optimism
+    tables:
+      - name: UniswapV3Factory_evt_PoolCreated
```

Comment:
i'm second guessing myself a bit now, after seeing this moved here. since `uniswap_v3_optimism` schema is called out but no tables specifically, i think it already accounts for all tables under that schema automatically. i'm afraid if we add this table specifically, it could break other models that relied on the auto assignment of all tables under that schema.

last check, can you remove this table completely (i.e. lines 5 & 6), and just let it run with only the schema there? would like to see if CI is able to run fine in that setup.

also do help me confirm if you are sticking with this decoded table vs. switching to the other spell of pools mentioned.

Reply:
Thank you for your guidance! I’ve decided to switch to using the uniswap_optimism_pools spell instead of the decoded table. This approach aligns better with the existing structure and ensures consistency across models.

Let me know if there’s anything additional I need to set up or confirm when using this spell as a source in my model. I’ll proceed with the adjustments and test the changes accordingly.

Reply:
i'm a bit confused. you mention moving to `uniswap_optimism_pools` spell as a source, but still see the old decoded table in the PR now?

you will need to list spell as a source here:
https://github.com/duneanalytics/spellbook/blob/main/sources/_subprojects_outputs/dex/_sources.yml

then update your model to read from `source('uniswap_optimism,' 'pools')`